share/exchange.yml

```diff
@@ -39,20 +39,20 @@ ASX_OTC:
   display_name: OTC Australia Stock Exchange
   market_times:
     dst:
-      daily_close: 20h
+      daily_close: 21h
       daily_open: -1h
       daily_settlement: 22h
       trading_breaks:
-      - - 4h30m
-        - 5h10m
+      - - 5h30m
+        - 6h10m
     partial_trading: {}
     standard:
-      daily_close: 21h
-      daily_open: 0s
+      daily_close: 20h
+      daily_open: -2h
```

Comment:
everything moved by an hour but this moved by `2h`?

Reply:
Ya. because the original trading hour also wrong.